### PR TITLE
Add Atom feeds

### PR DESCRIPTION
**Pull request by divergentdave**
This adds corresponding Atom feeds for any search request. They are linked to via &lt;link rel="alternate"/&gt; tags, and an icon at the bottom of each page of search results. The Atom feeds are paginated, using the appropriate Atom &lt;link&gt; tags.

### CODE REVIEW

File: app/helpers.js [hunk 1]
```diff
@@ -50,4 +50,9 @@ module.exports = {
     }
   },
 
+  // Remove form feed characters from a string
+  strip_ff: function(string) {
+    return string.replace(/\x0c/g, '');
+  }
```
Comment (konklone) [inline app/helpers.js]: :+1: Just mentally noting that perhaps we can remove this later, after we re-extract and re-index all the text.

File: app/routes.js [hunk 2]
```diff
@@ -50,6 +50,36 @@ module.exports = {
...
+        results: results,
+        query: req.query.query,
+        inspector: inspector,
+        page: page,
+        self_url: req.url
+      });
+    }, function(err) {
+      console.log("Noooo!");
+      res.status(500);
+      res.type("atom");
```
Comment (konklone) [inline app/routes.js]: This is really minor stuff, but I think the MIME type here would probably not be just `atom`. Definitely not worth holding up a merge over though.

File: views/reports.xml.ejs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<% } -%>
+<% if (results.hits.total > page * 10) { -%>
+<link rel="next" href="https://oversight.io/reports.xml?<%= query_string_next %>"/>
+<% } -%>
+<link rel="last" href="https://oversight.io/reports.xml?<%= query_string_last %>"/>
+<% var hits = results.hits.hits; -%>
+<% for (var i = 0; i < hits.length; i++) { -%>
+<% var report = hits[i]._source; -%>
+<% var highlight = hits[i].highlight; -%>
+<% var link = encodeURI('https:/' + '/oversight.io/report/' + report.inspector + '/' + report.report_id); -%>
```
Comment (konklone) [inline views/reports.xml.ejs]: why `'https:/' + '/oversight.io/report/'`?

**Comment by konklone** [comment]
Works perfectly!

File: app/routes.js (hunk 2 above)
Comment (divergentdave) [inline app/routes.js]: Indeed, but Express looks up MIME types from a dictionary, so "atom" gets turned into Content-Type:application/atom+xml; charset=utf-8

File: views/reports.xml.ejs (hunk 3 above)
Comment (divergentdave) [inline views/reports.xml.ejs]: I'm not entirely sure, but I think it was that http://... got interpreted as a line comment. Without that, I was getting bizarre compilation errors.

```
Unexpected token ILLEGAL in /home/pi/oversight.io/views/reports.xml.ejs while compiling ejs
```

File: app/routes.js (hunk 2 above)
Comment (konklone) [inline app/routes.js]: :boom:

**Pull request merged**